apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/fields/ValidatorField.jsx

```diff
@@ -54,26 +59,29 @@ const ValidatorField = ({
   };
 
   return (
-    <ValidatorsFieldContainer>
-      <Box p={1}>
-        <ScrollLoadingList
-          data={
-            showAll
-              ? validators
-              : [validators.find(v => v.validatorAddress === chosenVoteAccAddr) || validators[0]]
-          }
-          style={{ flex: showAll ? "1 0 256px" : "1 0 64px", marginBottom: 0, paddingLeft: 0 }}
-          renderItem={renderItem}
-          noResultPlaceholder={null}
-        />
-      </Box>
-      <SeeAllButton expanded={showAll} onClick={() => setShowAll(shown => !shown)}>
-        <Text color="wallet" ff="Inter|SemiBold" fontSize={4}>
-          <Trans i18nKey={showAll ? "distribution.showLess" : "distribution.showAll"} />
-        </Text>
-        <IconAngleDown size={16} />
-      </SeeAllButton>
-    </ValidatorsFieldContainer>
+    <>
+      {showAll && <ValidatorSearchInput noMargin={true} search={search} onSearch={onSearch} />}
+      <ValidatorsFieldContainer>
+        <Box p={1}>
+          <ScrollLoadingList
+            data={
+              showAll
+                ? validators
+                : [validators.find(v => v.validatorAddress === chosenVoteAccAddr) || validators[0]]
```

Comment:
calculating the data prop through a `useMemo` would help having to .find() again as well as keeping the array ref.